model/src/pyrenew/model/hospitalizations.py

```diff
@@ -62,7 +64,7 @@ def __init__(
             gen_int=gen_int,
             I0=I0,
             latent_infections=latent_infections,
-            observed_infections=None,
+            observed_infections=DeterministicVariable((0,)),
```

Comment:
[Mutable default](https://docs.python-guide.org/writing/gotchas/)

Reply:
But that's within the function's body, not the definition, so it should be OK, right? Also, is it everything a mutable default?  How about strings?

Reply:
I think the code is OK, @dylanhmorris

Reply:
Also, I changed a couple of instances where a `RandomVariable` had a default value to instead have None. Those cases with default strings I left them as strings are inmutable.

Reply:
Oops, I was tricked by how GH collapsed the diffs.